### PR TITLE
remove digest from operator

### PR DESCRIPTION
**Pull request by arturobrzut**

### CODE REVIEW

**Comment by arturobrzut** [comment]
The error in operator log if env variable will not be set
`{"level":"error","ts":1606920792.160643,"logger":"cmd","msg":"Operand environment variables not set correctly.","error":"ENV variable: OPERAND_LICENSING_IMAGE should have registry and image separated with \"/\" symbol","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/home/runner/go/pkg/mod/github.com/go-logr/zapr@v0.1.1/zapr.go:128\nmain.main\n\t/home/runner/work/ibm-licensing-operator/ibm-licensing-operator/cmd/manager/main.go:117\nruntime.main\n\t/opt/hostedtoolcache/go/1.15.5/x64/src/runtime/proc.go:204"}`

File: pkg/apis/operator/v1alpha1/helper.go [hunk 1]
```diff
@@ -383,6 +302,42 @@ func (container *Container) setImageParametersFromEnv(fullImageName string) erro
...
+	}
+	if container.ImageTagPostfix == "" {
+		container.ImageTagPostfix = temp.ImageTagPostfix
+	}
+	return nil
+}
+
+func CheckOperandEnvVar() error {
+	c := Container{}
+	if err := c.getImageParametersFromEnv("OPERAND_LICENSING_IMAGE"); err != nil {
```
Comment (adamdyszy) [inline pkg/apis/operator/v1alpha1/helper.go:325]: refactor OPERAND_LICENSING_IMAGE, and other operand images into variables, as they are used couple of times

File: pkg/apis/operator/v1alpha1/helper.go [hunk 2]
```diff
@@ -145,25 +131,8 @@ func (spec *IBMLicensingSpec) FillDefaultValues(isOCP4CertManager bool, isRouteE
...
-		if spec.ImageRegistry == "" {
-			spec.ImageRegistry = defaultQuayRegistry
-		}
-		if spec.ImageName == "" {
-			spec.ImageName = defaultLicensingImageName
-		}
-		if spec.ImageTagPostfix == "" {
-			spec.ImageTagPostfix = defaultLicensingImageTagPostfix
-		}
+	if err := spec.setContainer("OPERAND_LICENSING_IMAGE"); err != nil {
```
Comment (adamdyszy) [inline pkg/apis/operator/v1alpha1/helper.go:134]: refactor OPERAND_LICENSING_IMAGE, and other operand images into variables, as they are used couple of times

**Review by adamdyszy — CHANGES_REQUESTED** [review]

File: pkg/apis/operator/v1alpha1/helper.go (hunk 2 above)
Comment (arturobrzut) [reply pkg/apis/operator/v1alpha1/helper.go:134]: done

File: pkg/apis/operator/v1alpha1/helper.go (hunk 1 above)
Comment (arturobrzut) [reply pkg/apis/operator/v1alpha1/helper.go:325]: done

**Review by adamdyszy — APPROVED** [review]

**Pull request merged**